src/transaction/log_impl.h

```diff
@@ -800,6 +800,8 @@ extern int logpb_fetch_page (THREAD_ENTRY * thread_p, const LOG_LSA * req_lsa, L
 			     LOG_PAGE * log_pgptr);
 extern int logpb_copy_page_from_log_buffer (THREAD_ENTRY * thread_p, LOG_PAGEID pageid, LOG_PAGE * log_pgptr);
 extern int logpb_copy_page_from_file (THREAD_ENTRY * thread_p, LOG_PAGEID pageid, LOG_PAGE * log_pgptr);
```

Comment:
Shouldn't logpb_copy_page_from_file be moved to log_page_buffer.c now that it is replaced by logpb_read_page_from_file_or_page_server?

Reply:
Done